governance/pyth_staking_sdk/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@pythnetwork/staking-sdk",
+  "version": "0.0.0",
+  "description": "Pyth staking SDK",
+  "main": "src/index.ts",
+  "scripts": {
+    "build": "tsc",
+    "test": "jest"
```

Comment:
I can probably help with that  but also that's not so important, the important thing is to make sure that lint, prettier check, tsc, and unit test are being run as part of `test`, otherwise you won't get CI

Reply:
fixed.